mimic/model/clb_objects.py

```diff
@@ -13,6 +13,8 @@
 
 from six import string_types
 
+from toolz.dicttoolz import dissoc
+
```

Comment:
What is this library? Can we have an owner vet it or vouch for it? I'd rather not let any PR just pull in any dependencies without making sure they're good.